signify-verify

```diff
@@ -0,0 +1,22 @@
+#! /bin/sh
+
+# Copyright (C) 2022, 2023 - Tillitis AB
+# SPDX-License-Identifier: GPL-2.0-only
+
+# Use signify to verify a tkey-sign signature by hashing the file
+# content and checking the signature against the digest.
+# Usage:
+#
+# $ signify-verify MESSAGE PUBKEY
+
+if [ $# -ne 2 ]
+then
+    echo "Usage: signify-verify MESSAGE PUBKEY"
+    exit 1
+fi
+
+digestfile=$(mktemp /tmp/tkey-sign-digest.XXXXXX)
+sigfile="${1}".sig
+sha512sum $1 > ${digestfile}
```

Comment:
sha512sum is linux only. Do we want it running macos? 
We could use uname, or simply have an input..? 

But works great if I call `shasum -a 512 .. `